tools/cldr-code/src/main/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -368,21 +349,31 @@ public static PageId getPageId(String emoji) {
         final String major = getMajorCategory(emoji);
         final String minor = getMinorCategory(emoji);
         final PageId pageId = PageId.forString(major);
+        final Long minorOrder = minorToOrder.get(minor);
         switch (pageId) {
+            case Objects:
+                return (minorOrder < minorToOrder.get("money")) ? PageId.Objects : PageId.Objects2;
             case People:
-                if (minorToOrder.get(minor) < minorToOrder.get("person-fantasy")) {
-                    return PageId.People;
-                } else {
-                    return PageId.People2;
-                }
+                return (minorOrder < minorToOrder.get("person-fantasy"))
+                        ? PageId.People
+                        : PageId.People2;
             case Symbols:
-                if (minorToOrder.get(minor) < minorToOrder.get("transport-sign")) {
-                    return PageId.Symbols;
-                } else if (minorToOrder.get(minor) < minorToOrder.get("other-symbol")) {
-                    return PageId.Symbols2;
-                } else {
-                    return PageId.Symbols3;
-                }
+                return (minorOrder < minorToOrder.get("transport-sign"))
+                        ? PageId.Symbols
```

Comment:
I didn't catch this in the earlier PR. PageId.Symbols is *not* emoji, but rather for **numbers**:
```
    public enum PageId {
...
        Symbols(SectionId.Numbers),
```
eg https://cldr-staging.unicode.org/cldr-apps/v#/de/Symbols/

The symbol page IDs for emoji should be: PageId.Symbols2, Symbols3, Symbols4

(We might want to give these more meaningful names in a later PR.)

Reply:
Needs work. This needs to test against the UnicodeSets (as outlined) rather than the minor orders.